test/hotspot/jtreg/gc/shenandoah/generational/TestOldGrowthTriggers.java

```diff
@@ -58,14 +58,6 @@ public static void makeOldAllocations() {
                 int replaceIndex = r.nextInt(ArraySize);
                 int deriveIndex = r.nextInt(ArraySize);
                 switch (i & 0x3) {
-                    case 0:
-                        // 50% chance of creating garbage
-                        array[replaceIndex] = array[replaceIndex].max(array[deriveIndex]);
-                        break;
-                    case 1:
-                        // 50% chance of creating garbage
-                        array[replaceIndex] = array[replaceIndex].min(array[deriveIndex]);
-                        break;
```

Comment:
Could you have left this in place & reduced the heap size of the UseCompactHeaders test to half the current setting to induce the trigger?